libs/clist/clist_test.go

```diff
@@ -261,6 +261,8 @@ func TestWaitChan(t *testing.T) {
 			pushed++
 			time.Sleep(time.Duration(cmn.RandIntn(25)) * time.Millisecond)
 		}
+		// apply a deterministic pause so the counter has time to catch up
+		time.Sleep(time.Duration(25) * time.Millisecond)
```

Comment:
```suggestion
		time.Sleep(25 * time.Millisecond)
```

Reply:
Ideally, we wouldn't use a timer but some channel to signal that the counter actually caught up. But given that this test fails quite often (and we already use sleep with a random timeout 🙈), it's probably worth trying this out 👍 
Thanks @kevlubkcm!

Reply:
yup. was thinking that too but simple change is simple :D